eth/tracers/native/prestate.go

```diff
@@ -45,6 +45,10 @@ type account struct {
 	Storage map[common.Hash]common.Hash `json:"storage,omitempty"`
 }
 
+func (a *account) isExists() bool {
```

Comment:
Nitpick: to stay consistent rename to `empty` as in stateObject and change accordingly:

https://github.com/ethereum/go-ethereum/blob/d86fe26f67d8ad17d8b66da9a8908101fb4c11cb/core/state/state_object.go#L98-L100

Also note the `balance.Sign() == 0` check. Balance < 0 should not happen but it still makes account non-empty.